source/content/object-cache.md

```diff
@@ -128,19 +128,99 @@ All plans except for the Basic plan can use Object Cache. Sandbox site plans can
+    // $conf['cache_backends'][] = 'sites/all/modules/contrib/redis/redis.autoload.inc';
+    $conf['cache_default_class'] = 'Redis_CacheCompressed';
+    $conf['cache_prefix'] = array('default' => 'pantheon-redis');
+    // Do not use Redis for cache_form (no performance difference).
+    $conf['cache_class_cache_form'] = 'DrupalDatabaseCache';
+    // Use Redis for Drupal locks (semaphore).
+    $conf['lock_inc'] = 'sites/all/modules/redis/redis.lock.inc';
+    // Or if you've installed the redis module in a contrib subdirectory, use:
+    // $conf['lock_inc'] = 'sites/all/modules/contrib/redis/redis.lock.inc';
+  }
+  ```
+
+1. Enable the module via from `/admin/modules` if you haven't already done so with Terminus.
+
+1. Verify Redis is enabled by going to the Dashboard and clicking **Connection Info**. If you see the Redis cache connection string, Redis is enabled.
+
+1. Visit `/admin/config/development/performance/redis` and open **Connection Information** to verify the connection.
+
+<Accordion title="Database Cleanup (optional)" id="database-cleanup-d7" icon="lightbulb">
+
+After enabling Redis, there are cache tables in the database that are no longer being used. Even when the Drupal cache is cleared, these tables will not be emptied. For sites that were live for awhile before Redis was enabled, there could be significant amounts of data in these tables. Removing this data could increase the speed of cloning, exporting and backing up the database.
+
+1. [Connect directly to MySQL](/mysql-access) and run the command below to view the cache:
+
+  ```sql
+  SELECT table_name FROM information_schema.tables WHERE table_name LIKE 'cache%' AND table_name != 'cache_form';
+  ```
+
+ This returns a list of all the cache tables in the database. These are safe to empty, but don't remove the tables themselves in case Redis is disabled in the future.
+
+1. Run the command below on each table, replacing `<tablename>` with the name of the cache table, to empty the cache:
+
+  ```sql
+  TRUNCATE TABLE `<tablename>`;
+  ```
+
+</Accordion>
+
+</Tab>
+
+
 <Tab title="Drupal 8" id="d8-install">
 
-1. Enable Object Cache from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**. It may take a couple minutes for Object Cache to come online.
+1. Enable Object Cache in your Pantheon Site Dashboard under **Settings** > **Add Ons** > **Add**. It may take a couple minutes for Object Cache to come online.
 
-1. Install and activate the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
+1. Download and activate the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
 
   <Alert title="Note" type="info">
 
   You **must** activate the module before proceeding.
 
   </Alert>
 
-  You can install and enable the module from the command line using [Terminus](/terminus):
+1. Run `composer require drupal/redis` in the local command line to install Redis.
```

Comment:
After installing locally, it is necessary to commit and push the change to Pantheon:
```
git commit --am "Add drupal/redis dependency"
git push origin master
```
This must be done before running the `terminus drush` command below. The Drush command might fail if the build started by the `git push` has not finished yet.

Reply:
That would go above the commit and push, of course.